packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,278 @@
+# Terra Application Layout
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
+
+## Props
+
```

Comment:
nit: I'd sort the props below alphabetically. I think it makes it easier to find the prop you are looking for.

Reply:
Should we update our lint guide to reflect this? In the past we did two groups alphabetically: required and optional